### PR TITLE
Release notes 10.4.2

### PR DESCRIPTION
**Pull request by swethakundaram**

### CODE REVIEW

File: learn/wavemaker-release-notes/v10-4-2.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+sidebar_label: "Current v10.4.2"
+---
+*WaveMaker released a new update version with bug fixes.*
+
+## Bug fixes
+
+---
+
+- Supporting API keys to pass in header/query for endpoints in an imported OpenAPI doc.
+- The API keys are profiled and can be configured with different values for different environments (dev/stage/prod).
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-4-2.md:13]: The above two are features and not bug fixes

File: learn/wavemaker-release-notes.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ WaveMaker is constantly working on bringing new features and improvements to the
 
 |Version Update|WMO details| WMO: Released on |WME details| WME: Released on |
 |---|---|---|---|---|
+|[WaveMaker v10.4.2](/learn/wavemaker-release-notes/v10-4-2)|Bug fixes | 07 May 2020 | -|-|
```
Comment (vibhus) [inline learn/wavemaker-release-notes.md:22]: It has Feature + Bug fixes

**Comment by prasanth-reddy-wavemaker-com** [comment]
I am good if @vibhus is good with this.

File: learn/wavemaker-release-notes.md (hunk 2 above)
Comment (swethakundaram-wm) [reply learn/wavemaker-release-notes.md]: resolved

**Pull request merged**